tests/schedule.go

```diff
@@ -244,6 +245,7 @@ func (s *ScheduleFunctionalSuite) TestBasics() {
 	s.Nil(describeResp.SearchAttributes.IndexedFields[searchattribute.BinaryChecksums])
 	s.Nil(describeResp.SearchAttributes.IndexedFields[searchattribute.BuildIds])
 	s.Equal(schMemo.Data, describeResp.Memo.Fields["schedmemo1"].Data)
+	fmt.Printf("StartWorkflowAction: %x", describeResp)
```

Comment:
What about checking `type` here instead of a printf? (as suggested in https://github.com/temporalio/temporal/pull/5353/files/abca448f2f65d1ed395552f4e95155a6fa71b190..660d2237903e0c6942e9d01c0483c941ab12ef36#r1472055214)